primitives/src/crh/poseidon/parameters/mnt4753.rs

```diff
@@ -270,17 +290,13 @@ impl PoseidonParameters for MNT4753PoseidonParameters {
         field_new!(Fr,BigInteger([0xbeaefc0f3ffffe15,0x31fa60c505c528da,0x17062bf32a5800cc,0x7f6c8a491e1c85d8,0x51c275d51d6bb509,0xca2b3dc7bc07b33e,0xfbde52978687148a,0xac5de44ad3586169,0x5544299cb8c3db5f,0x244ac8e0636993bb,0xdb58cffd2ff83d0,0x1120aca75573a,])),
     ];
 
-    // The MDS matrix supporting short Montgomery multiplication
-    const MDS_CST_SHORT: &'static[Fr]  = &[
-        // These entries are in Montgomery representation with respect to R2 = 2^64.
-        field_new!(Fr,BigInteger([0x1b06b82936573768, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0xa8a66953a924365d, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0xb412c015510c2717, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0x351fdbd63ac0afdb, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0x302be8e2c8e27f02, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0x7dcdc338f53308c, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0x5220f8b41dab7db4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0x524543d141024c82, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-        field_new!(Fr,BigInteger([0x3657a2432f363f4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])),
-    ];
+    /// Short Montgomery multiplication with respect to the short Montgomery constant R_2=2^64
```

Comment:
Sorry for giving the wrong instructions here: Let us write "Uses short Mongomery multiplication with respect to..."